package/MDAnalysis/core/universe.py

```diff
@@ -658,14 +659,12 @@ def as_Universe(*args, **kwargs):
         return args[0]
     return Universe(*args, **kwargs)
 
-asUniverse = deprecate(as_Universe, old_name='asUniverse', new_name='as_Universe')
```

Comment:
I *think* it was meant to be something like `np.asarray`, so you could pass either a filename or Universe...